### PR TITLE
Add spec in uploader.json for mysqladmin_vars

### PR DESCRIPTION
**Pull request by JoySnow**

### CODE REVIEW

**Comment by xiangce** [comment]
Looks good to me

File: uploader.json [hunk 1]
```diff
@@ -4484,6 +4489,15 @@
...
         },
+        "mysqladmin_vars": {
+            "host": [
+                {
+                    "archive_file_name": "/insights_commands/mysqladmin_variables",
+                    "command": "/bin/mysqladmin variables",
+                    "pattern": []
+                }
+            ]
+        },
```
Comment (bfahr) [inline uploader.json]: @JoySnow please remove this section of the json as it is not being used.  If you don't mind, could you also add the command to `uploader.v2.json` in your PR?

**Comment by eduardocerqueira** [comment]
test this please

**Comment by bfahr** [comment]
This PR can be closed.  I went ahead and made the required changes in PR #61

**Comment by eduardocerqueira** [comment]
test this please

**Comment by JoySnow** [comment]
Due to PR https://github.com/RedHatInsights/insights-frontend-assets/pull/61 is the proper replacement,
close this one here.

**Pull request closed without merging**